Tests/ConcordiumSwiftSDKTests/ClientTests.swift

```diff
@@ -18,38 +19,60 @@ final class ClientTests: XCTestCase {
     func testClientGetCryptographicParameters() async throws {
         let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
         defer {
-          try! group.syncShutdownGracefully()
+            try! group.syncShutdownGracefully()
         }
         let channel = try GRPCChannelPool.with(
-            target: channelTarget,
-            transportSecurity: .plaintext,
-            eventLoopGroup: group
+                target: channelTarget,
+                transportSecurity: .plaintext,
+                eventLoopGroup: group
         )
         defer {
-          try! channel.close().wait()
+            try! channel.close().wait()
         }
         let client = Client(channel: channel)
         let hash = try BlockHash(fromHexString: someBlockHash)
         let res = try await client.getCryptographicParameters(at: .hash(hash))
         print(res)
     }
-    
+
     func testClientGetNextAccountSequenceNumber() async throws {
         let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
         defer {
-          try! group.syncShutdownGracefully()
+            try! group.syncShutdownGracefully()
         }
         let channel = try GRPCChannelPool.with(
-            target: channelTarget,
-            transportSecurity: .plaintext,
-            eventLoopGroup: group
+                target: channelTarget,
+                transportSecurity: .plaintext,
+                eventLoopGroup: group
         )
         defer {
-          try! channel.close().wait()
+            try! channel.close().wait()
         }
         let client = Client(channel: channel)
         let addr = try AccountAddress(base58Check: someAccountAddress)
         let res = try await client.getNextAccountSequenceNumber(of: addr)
         print(res)
     }
+
+    func testClientGetAccountInfo() async throws {
+        let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+        defer {
+            try! group.syncShutdownGracefully()
+        }
+        let channel = try GRPCChannelPool.with(
+                target: channelTarget,
+                transportSecurity: .plaintext,
+                eventLoopGroup: group
+        )
+        defer {
+            try! channel.close().wait()
+        }
+        // TODO: Error when something goes wrong is completely useless ("missing field").
```

Comment:
Is there anything we can do about that (i.e., should we create a jira), or is there something that the Blockchain team should do about that, or will it never happen in real life.

Reply:
I think it's an issue with the generated client. But haven't dug into it. I'll create a task.

Reply:
Filed https://concordium.atlassian.net/browse/CBW-1641. Will remove the TODO.